src/utils/get-link-label.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Create an accessible link text for an author's social links and contact details.
+ *
+ * Example:
+ * <a aria-label={getLinkLabel(name, link)} href='twitter.com/author'>Twitter</a>
+ *
+ * @param name
+ * @param link
+ */
+export const getLinkLabel = (
+  name: string,
+  link: { url: string; title: string }
+) => {
+  const findMatch = link.url.match(
```

Comment:
This list is unlikely to change but it's worth mentioning that testing against a regex which is separate to linkMap's keys needs both parts updating. It's always better to work from a single source to avoid bugs.

One way of doing this (and this is similar to the funding scheme listing page) would be to use `Object.keys` and then `Array.find` to return a matching value

https://codepen.io/creido-welly/pen/JjORWoG?editors=0011

```
Object.keys(linkMap).find(key => str.includes(key))
```

As an aside, I think you should also include the colons for mailto: and tel: in the keys (I've done this in the codepen)

Reply:
Good point. I will update.